client/my-sites/themes/index.node.js

```diff
@@ -35,7 +35,7 @@ export default function( router ) {
 			loggedOut,
 			makeLayout
 		);
-		router( '/themes/upload/*', makeLayout );
+		router( '/themes/upload*', makeLayout );
```

Comment:
I like to keep them explicit (to disallow `/themes/uploadfoobar` -- something like that bit me recently with a v4 legacy route :roll_eyes:):

```js
router( [ '/themes/upload', '/themes/upload/*' ], makeLayout );
```